.github/workflows/cypress-firefox.yml

```
@@ -0,0 +1,22 @@
+name: Cypress push and pull
+
+on: [push, pull_request]
+
+jobs:
+  cypress-run:
+     name: firefox
+     runs-on: ubuntu-18.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: update test fixture
+        run: npm run pre:cy:genfix
+
+      - name: Cypress run
+        uses: cypress-io/github-action@v2
+        with:
+          start: npm run serve
```

The `serve` command seems like it just serves whatever's in the `build` directory, but doesn't create it.

We could use the `start` command and and timeout here like you already have:

```suggestion
          start: yarn start
```

Or we could use a build step first:

```
          build: yarn build
          start: yarn start
```

If we have a build step first, we can probably remove the `wait-on-timeout` line.